### ISSUE TITLE
More translatable strings

### ISSUE THREAD

**Issue by TWME-TW** [kangarko/Boss#1085]
# More translatable strings

## How it would benefit others?
Make it more intuitive for non-English speakers to use this plugin

## What would happen if we won't implement this feature? How's your server being affected?
Maybe some staff will use it harder

## If suggesting something new - Can the old system be modified/improved instead?
Yes

## Problems encountered:
When I finished the translation and entered the game, using the `/boss menu`, I found that there were still many untranslated strings. After checking, I confirmed that I have finished translating all non-comments. This is the translation file:
https://github.com/TWME-TW/twme_translate/blob/main/Boss/localization/messages_boss_tw.yml

**Comment by kangarko** [comment]
Thank you! Right now we only allow translating strings for end users, our menus are intended for administrators only.

**Comment by TWME-TW** [comment]
> Thank you! Right now we only allow translating strings for end users, our menus are intended for administrators only.

Will this be added to the Official Roadmap?

**Comment by kangarko** [comment]
I can consider adding this in the future yes!

**Comment by TWME-TW** [comment]
I just found that you added my translation to the official translation, but I still have some words and punctuation to correct. The translation I posted is just to prove that I have indeed translated all the words that can be translated, and there are no formatting errors.

I used google translate to write this article (of course, my translation is not), so maybe it caused some misunderstanding, I just want to fully translate all the strings of the plugin, not to submit my translation, if I want to submit my translation  A special ISSUES will be officially opened one day. If it caused you trouble, I am very sorry, and I apologize again.
